utils/check_repo.py

```diff
@@ -103,6 +103,7 @@
 # should **not** be the rule.
 IGNORE_NON_AUTO_CONFIGURED = PRIVATE_MODELS.copy() + [
     # models to ignore for model xxx mapping
+    "ViTMAEForPreTraining",
```

Comment:
Why is this not added to the `ForPreTraining` auto-mapping? This one does not have a unified API so every model should be in it.

Reply:
Ok, I'll add it.